cloudbuild-dev.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2019 The Forseti Security Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Purpose - This Google Cloudbuild configuration defines the steps
```

Comment:
Would it make sense to organize these cloudbuild scripts better?  Such as a `/build` directory?

Reply:
We can but the cloudbuild-dev.yaml and cloudbuild-release.yaml files in a build folder, sure.  The "cloudbuild.yaml" file will need to remain in the root if it is to replace TravisCI.

Reply:
Nevermind then, I think it's good to keep them all together.

Reply:
already moved.  I think putting them in a folder makes sense.  Keeps the root no more cluttered than needed.